exercises/phone-number/canonical-data.json

```diff
@@ -84,6 +84,21 @@
         "error": "more than 11 digits"
       }
     },
+    {
+      "uuid": "4a1509b7-8953-4eec-981b-c483358ff531",
```

Comment:
I believe existing test cases must not be changed, a new UUID is needed for a reimplementation.

Reply:
Something seems strange, then, since c6a5f007-895a-4fc5-90bc-a7e70f9b5cad was also reimplemented with a test concerning 9 numbers.

```
exercises/phone-number/canonical-data.json:47-    {
exercises/phone-number/canonical-data.json:48-      "uuid": "2de74156-f646-42b5-8638-0ef1d8b58bc2",
exercises/phone-number/canonical-data.json:49:      "reimplements": "c6a5f007-895a-4fc5-90bc-a7e70f9b5cad",
exercises/phone-number/canonical-data.json:50-      "description": "invalid when 9 digits",
exercises/phone-number/canonical-data.json:51-      "comments": ["Make incorrect number of digit errors consistent."],
exercises/phone-number/canonical-data.json:52-      "property": "clean",
--
exercises/phone-number/canonical-data.json:89-    {
exercises/phone-number/canonical-data.json:90:      "uuid": "c6a5f007-895a-4fc5-90bc-a7e70f9b5cad",
exercises/phone-number/canonical-data.json:91-      "description": "invalid when more than 11 digits",
exercises/phone-number/canonical-data.json:92-      "property": "clean",
--
exercises/phone-number/canonical-data.json:100-    {
exercises/phone-number/canonical-data.json:101-      "uuid": "4a1509b7-8953-4eec-981b-c483358ff531",
exercises/phone-number/canonical-data.json:102:      "reimplements": "c6a5f007-895a-4fc5-90bc-a7e70f9b5cad",
exercises/phone-number/canonical-data.json:103-      "description": "invalid when more than 11 digits",
```

Reply:
That's a mistake on my part relating to the latest push.